ega-data-api-key/src/main/java/eu/elixir/ega/ebi/keyproviderservice/KeyProviderServiceApplication.java

```diff
@@ -15,29 +15,21 @@
  */
 package eu.elixir.ega.ebi.keyproviderservice;
 
-import com.google.common.cache.CacheBuilder;
 import org.springframework.boot.SpringApplication;
-import org.springframework.boot.autoconfigure.EnableAutoConfiguration;
-import org.springframework.boot.autoconfigure.SpringBootApplication;
-import org.springframework.boot.autoconfigure.jdbc.DataSourceAutoConfiguration;
-import org.springframework.boot.autoconfigure.orm.jpa.HibernateJpaAutoConfiguration;
-import org.springframework.cache.CacheManager;
 import org.springframework.cache.annotation.EnableCaching;
-import org.springframework.cache.guava.GuavaCache;
-import org.springframework.cache.support.SimpleCacheManager;
 import org.springframework.cloud.client.discovery.EnableDiscoveryClient;
+import org.springframework.cloud.netflix.eureka.EnableEurekaClient;
 import org.springframework.cloud.netflix.hystrix.EnableHystrix;
 import org.springframework.cloud.netflix.hystrix.dashboard.EnableHystrixDashboard;
-import org.springframework.context.annotation.Bean;
-import springfox.documentation.swagger2.annotations.EnableSwagger2;
+import org.springframework.context.annotation.ComponentScan;
+import org.springframework.context.annotation.Configuration;
 
-import java.util.Collections;
-import java.util.concurrent.TimeUnit;
+import springfox.documentation.swagger2.annotations.EnableSwagger2;
 
-@SpringBootApplication
+@Configuration
+@ComponentScan
 @EnableCaching
 @EnableHystrix
-@EnableAutoConfiguration(exclude = {DataSourceAutoConfiguration.class, HibernateJpaAutoConfiguration.class})
 @EnableSwagger2
```

Comment:
This annotation is also in SwaggerConfig in config package so I would remove it from here